formbuilder/routes.js

```diff
@@ -17,7 +17,7 @@ import Check from "./components/Check";
 
 const common = {
   notifications: NotificationContainer,
-  fieldList: FieldListContainer,
+  sidebarComponent: FieldListContainer,
```

Comment:
I'm sorry, I don't understand the direction you want to take. Would you mind showing me what you have in mind with some code? (I just see another level of indirection without so much value so I'm probably missing your point)